src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java

```diff
@@ -208,6 +217,11 @@ public String renderPrimitiveValue(Schema schema, String value) {
     throw new IllegalArgumentException("Schema is of unknown type.");
   }
 
+  @Override
+  public String renderValueAsString(String value) {
```

Comment:
Would be nice to replace calls to this with something like, `renderPrimitiveValue(Bla.STRING, value)`, but it looks like it's much harder with the current method signature of `renderPrimitiveValue`.

Reply:
I agree, it was just a black hole of type wrapping I didn't want to get into.

Reply:
A possible approach is to create a static field `STRING` in `Schema` which contains a `Schema` of type `STRING`. Up to you if you want to do that.